bin/autodetect/Main.cc

```diff
@@ -83,7 +84,8 @@ int main(int argc, char** argv) {
         ml::counter_t::E_TSADNumberMemoryLimitModelCreationFailures,
         ml::counter_t::E_TSADNumberPrunedItems,
         ml::counter_t::E_TSADAssignmentMemoryBasis,
-        ml::counter_t::E_TSADOutputMemoryAllocatorUsage};
+        ml::counter_t::E_TSADOutputMemoryAllocatorUsage,
+        ml::counter_t::E_TSADMaxResidentSetSize};
```

Comment:
Does reporting only the **maximum** resident set size make sense? Can we also collect the **current** resident set size?